streams/shared/src/main/scala/zio/stream/StreamEffect.scala

```diff
@@ -320,6 +322,25 @@ private[stream] object StreamEffect extends Serializable {
       }
     }
 
+  final def fromInputStream(is: InputStream): StreamEffect[Any, Nothing, Byte] =
+    StreamEffect[Any, Nothing, Byte] {
+      Managed.effectTotal {
```

Comment:
This is not a total effect, I can see `Managed.makeEffect` which is probably more appropriate here.

Reply:
Well the `Managed` only returns a thunk that, so in that regard it *should* be total

Reply:
This is how we treated all `StreamEffect` code so far.